src/main/java/com/minecolonies/coremod/colony/managers/EventStructureManager.java

```diff
@@ -97,7 +93,7 @@ public boolean spawnTemporaryStructure(
         final Level world = colony.getWorld();
 
         final int y = BlueprintTagUtils.getNumberOfGroundLevels(structure, 4) - 1;
-        final BlockPos spawnPos = targetSpawnPoint.offset(0, -y, 0).offset(structure.getPrimaryBlockOffset());
+        final BlockPos spawnPos = targetSpawnPoint.below(y).above(structure.getPrimaryBlockOffset().getY());
```

Comment:
wasn't this on purpose to make the main spawn-point the primary offset of the structure?

Reply:
That was probably the intent, but that's what my version _actually_ does.  The schematic loads around its anchor point anyway, so we just need to line up the `y` properly and it's already correct.

Reply:
ahhh, ok